src/main/java/ua/com/juja/microservices/keepers/slackbot/dao/impl/RestKeeperRepository.java

```diff
@@ -20,4 +30,30 @@
     public RestKeeperRepository(RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
     }
+
+    @Value("${keepers.baseURL}")
+    private String urlBase;
+    @Value("${endpoint.getKeeperDirections}")
+    private String urlGetKeeperDirections;
+
+    @Override
+    public List<String> getKeeperDirections(String uuid) {
+        HttpEntity<String> request = new HttpEntity<>(uuid, setupBaseHttpHeaders());
+        List<String> result;
```

Comment:
You should initialize result by empty List. Otherwise, in some situations, you can get Null Pointer Ex